.changes/tinymce/7.1.1.md

```diff
@@ -0,0 +1,8 @@
+## 7.1.1 - 2024-05-20
+
+### Fixed
+- Insert/Edit image dialog lost focus after the image upload completed. #TINY-10885
+- Deleting into a list from a paragraph that has an `img` tag could cause extra inline styles to be added. #TINY-10892
+- Resolved an issue where emojis were not loading correctly due to a broken CDN. #TINY-10878
```

Comment:
```suggestion
- Resolved an issue where emojis configured with the `emojiimages` database were not loading correctly due to a broken CDN. #TINY-10878
```

Although even this is misleading, because any existing emoji images in content will be broken and I don't think the plugin will fix them automatically.